documentation/wiki/ChangeWaves.md

```diff
@@ -27,6 +27,7 @@ A wave of features is set to "rotate out" (i.e. become standard functionality) t
 - [AppDomain configuration is serialized without using BinFmt](https://github.com/dotnet/msbuild/pull/9320) - feature can be opted out only if [BinaryFormatter](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.serialization.formatters.binary.binaryformatter) is allowed at runtime by editing `MSBuild.runtimeconfig.json`
 - [Warning on serialization custom events by default in .NET framework](https://github.com/dotnet/msbuild/pull/9318)
 - [Cache SDK resolver data process-wide](https://github.com/dotnet/msbuild/pull/9335)
+- [Target parameters will be unquoted, this will remove the possibility to have ';' in target name ](https://github.com/dotnet/msbuild/pull/9452)
```

Comment:
NIT: it not precise. You can still have target with ";" as before, but you cannot specify it in /t: parameter anymore. Such target would still be invoked by msbuild logic by "BeforeTarget" or "AfterTarget".